provenance-verifier/pkg/workflow.go

```diff
@@ -27,9 +27,27 @@ func WorkflowFromBytes(content []byte) (*Workflow, error) {
 	return &self, nil
 }
 
-func (w *Workflow) validateTopLevelEnvironmentVariables() error {
-	if w.workflow.Env != nil && len(w.workflow.Env.Vars) > 0 {
-		return fmt.Errorf("%w", errorTopLevelEnvVariables)
+func (w *Workflow) validateTopLevelEnv() error {
+	return validateEnv(w.workflow.Env, "top level")
```

Comment:
you could use an errors.Wrap(err, "top level") and not need to propogate msg strings here and for job-level

Reply:
great idea. I will!

Reply:
mhh, actually Wrap is not available from the standard errors package https://pkg.go.dev/errors. It requires this additional package https://github.com/pkg/errors

I'll merge as-is. If you think we should use Wrap in the future, let's create a tracking issue for it.